spec/features/personas/sign_in_as_a_claims_user_persona_spec.rb

```diff
@@ -50,6 +54,10 @@ def given_there_is_an_existing_claims_persona_for(persona_name)
   create(:persona, persona_name.downcase.to_sym, service: "claims")
 end
 
+def and_given_i_have_a_school
+  create(:gias_school)
+end
```

Comment:
I think users will have a `School`, not a `GiasSchool`. You are likely to have started this before the `School` model and relationship with ` `GiasSchool` was in place.